apps/lockfile-explorer-web/src/containers/LockfileEntryDetailsView/index.tsx

```diff
@@ -36,14 +38,22 @@ export const LockfileEntryDetailsView = (): JSX.Element | ReactNull => {
 
   const selectResolvedEntry = useCallback(
     (dependencyToTrace) => () => {
-      if (inspectDependency && inspectDependency.name === dependencyToTrace.name) {
+      if (inspectDependency && inspectDependency.entryId === dependencyToTrace.entryId) {
         if (dependencyToTrace.resolvedEntry) {
           dispatch(pushToStack(dependencyToTrace.resolvedEntry));
         } else {
           logDiagnosticInfo('No resolved entry for dependency:', dependencyToTrace);
         }
       } else if (selectedEntry) {
+        console.log('dependency to trace: ', dependencyToTrace);
         setInspectDependency(dependencyToTrace);
+
+        // Check if we need to calculate influencers.
+        // If the current dependencyToTrace is a peer dependency then we do
+        if (dependencyToTrace.dependencyType !== IDependencyType.PEER_DEPENDENCY) {
```

Comment:
Since `IDependencyType` is an enum, it should be called `DependencyType`.  The "I" prefix is only used for `interface` types